.gitignore

```diff
@@ -9,7 +9,7 @@
 *.idea
 
 # Build
-packages/Debian/build/
```

Comment:
we need to be consistent with this, what was the issue with the path `packages/Debian/build/`

Reply:
As we have supported all architecture (i.e directory structure changed) so Packages are being stored in `/foglamp-south-sinusoid/packages/build`